### PR TITLE
Add support for MNK60

### PR DESCRIPTION
**Pull request by kopibeng**
Add support for MNK60

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/kopibeng/mnk60/info.json
Comment (waffle87) [inline keyboards/kopibeng/mnk60/info.json:1]: Please run `qmk format-json path/to/info.json` on this file.

File: keyboards/kopibeng/mnk60/info.json [hunk 1]
```diff
@@ -0,0 +1,989 @@
...
+	"indicators": {
+		"caps_lock": "D4"
+	},
+	"ws2812": {
+		"pin": "F6"
+	},
+	"rgblight": {
+		"pin": "F6",
+		"led_count": 14,
+		"hue_steps": 8,
```
Comment (waffle87) [inline keyboards/kopibeng/mnk60/info.json:36]:
```suggestion
		"led_count": 14,
```

File: keyboards/kopibeng/mnk60/keymaps/via/keymap.c [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+	),
+	
+	// Fn3 Layer
+	[3] = LAYOUT_all(
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,          KC_TRNS,          KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS
+	),
```
Comment (waffle87) [inline keyboards/kopibeng/mnk60/keymaps/via/keymap.c:55]:
```suggestion
	)
```

File: keyboards/kopibeng/mnk60/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+	),
+	
+	// Fn3 Layer
+	[3] = LAYOUT_all(
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,          KC_TRNS,          KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS
+	),
```
Comment (waffle87) [inline keyboards/kopibeng/mnk60/keymaps/default/keymap.c:55]:
```suggestion
	)
```

**Review by waffle87 — CHANGES_REQUESTED** [review]

**Comment by waffle87** [comment]
I've marked a comment as unresolved as the suggested change was not made. Make sure you actually replace the content of `info.json` file with the output of the `format-json` command, as simply running the command does not do anything.

**Comment by kopibeng** [comment]
Done. Please let me know if there is any issue 😄

**Review by waffle87 — APPROVED** [review]

**Review by dunk2k — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**